src/module-1/card/index-style-card.css

```diff
@@ -0,0 +1,95 @@
+body {
+    background-color: #F9F9F9;
+}
+
```

Comment:
no empty lines
or insert it before every selector to keep the same code style for the whole file